pyiron_contrib/atomistics/runner/job.py

```diff
@@ -1,235 +1,444 @@
+    - RunnerStructureContainer (StructureStorage): Storage container for
+                                                   RuNNer training datasets.
+
+FIXME:
+At this point, RunnerFit heavily relies on an unpublished version of an ASE
+calculator for RuNNer.
+
+Reference:
+    - RuNNer online documentation](https://theochem.gitlab.io/runner)
 """
 
-import os.path
-from glob import glob
+import numpy as np
+
+from ase import Atoms
+import ase.io.runner.runner as io
+from ase.calculators.runner.runner import Runner, DEFAULT_PARAMETERS
+from ase.calculators.runner.runnersinglepoint import RunnerSinglePointCalculator
 
 from pyiron_base import state, GenericJob, Executable, DataContainer
+from pyiron_atomistics.atomistics.structure.structurestorage import StructureStorage
+from pyiron_atomistics.atomistics.structure.atoms import pyiron_to_ase, ase_to_pyiron
 
-import numpy as np
-import pandas as pd
+__author__ = 'Alexander Knoll'
+__copyright__ = 'Copyright 2021, Georg-August-Universität Göttingen - Behler '\
+                'Group'
+__version__ = '0.1'
+__maintainer__ = 'Alexander Knoll'
+__email__ = 'alexander.knoll@chemie.uni-goettingen.de'
+__status__ = 'development'
+__date__ = 'January 7, 2022'
+
+
+class PotentialFittingBase(GenericJob):
+    """FIXME: Class copied from AtomicRex in pyiron_contrib."""
+
+    def __init__(self, project, job_name):
+        """Initialize the class.
+
+        Args:
+            project (FIXME): The project container in which the job is created.
+            job_name (str):  The label of the job (used for all directories).
+        """
+        super().__init__(project, job_name)
+
+
+class RunnerInput(DataContainer):
+    """Extend the DataContainer class for future customization."""
 
-__author__ = "Marvin Poul"
-__copyright__ = "Copyright 2020, Max-Planck-Institut für Eisenforschung GmbH - " \
-                "Computational Materials Design (CM) Department"
-__version__ = "0.1"
-__maintainer__ = "Marvin Poul"
-__email__ = "poul@mpie.de"
-__status__ = "development"
-__date__ = "March 3, 2021"
+    __version__ = '0.3.0'
+    __hdf_version__ = '0.3.0'
 
-AngstromToBohr = 1.88972612
-ElectronVoltToHartree = 0.03674932218
+    def __init__(self, *args, **kwargs):
+        """Initialize the class."""
+        super(RunnerInput, self).__init__(*args, **kwargs)
 
 
-class RunnerFit(GenericJob):
+class RunnerStructureContainer(StructureStorage):
+    """Store chemical structures as a Runner training dataset."""
+
+    __version__ = '0.3.0'
+    __hdf_version__ = '0.3.0'
+
     def __init__(self, project, job_name):
-        super().__init__(project, job_name) 
-        self._training_ids = []
+        """Initialize the class.
+
+        Args:
+            project (FIXME): The project container in which the job is created.
+            job_name (str):  The label of the job (used for all directories).
+        """
+        super(RunnerStructureContainer, self).__init__()
+
+    def append(self, structure):
```

Comment:
I opened a merge request [here](https://github.com/pyiron/pyiron_atomistics/pull/518). If accepted, all properties needed for RuNNer fits (energy, forces, atomic charges), **except the totalcharge**,  would be automatically stripped from an ASE Atoms object. The total charge is dealt with separately in the `RunnerFit` class, because it is not a standard property in ASE calculators.